src/signals/settings/categories/components/Detail.test.tsx

```diff
@@ -5,6 +5,7 @@ import userEvent from '@testing-library/user-event'
 import * as reactRedux from 'react-redux'
 import * as reactRouterDom from 'react-router-dom'
 
+import * as useConfirm from 'hooks/useConfirm'
```

Comment:
import ESLint error

Reply:
what ESLint error are you referring to? Because I only get greyed out notifications, which are ignored. 
![image](https://github.com/Amsterdam/signals-frontend/assets/25867841/ec3370e8-b267-4f7c-9415-535e09cf2ab0)

Reply:
<img width="1587" alt="Screenshot 2023-08-08 at 08 11 06" src="https://github.com/Amsterdam/signals-frontend/assets/46756331/f4befaa8-ace9-4ba6-8bb7-cd230fca4acc">

Reply:
Oh no, I do get that notification. Was looking in a different file. Thanks for the quick reply.